dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/enums/DbType.java

```diff
@@ -38,8 +37,9 @@ public enum DbType {
     DB2(7, "db2"),
     PRESTO(8, "presto"),
     H2(9, "h2"),
-    REDSHIFT(10, "redshift"),
-    ATHENA(11, "athena"),
+    REDSHIFT(10,"redshift"),
+    ATHENA(11,"athena"),
```

Comment:
Please don't change these code. 
```suggestion
    REDSHIFT(10, "redshift"),
    ATHENA(11, "athena"),
```

Reply:
done.